src/controller/CommissioningDelegate.h

```diff
@@ -340,6 +353,19 @@ class CommissioningParameters
         return *this;
     }
 
+    CommissioningParameters & SetRequireTermsAndConditionsAcknowledgement(bool requireTermsAndConditionsAcknowledgement)
```

Comment:
This API is confusing. Isn't it the device that *requires* (or not) TC acknowledgement or not, and this parameter controls whether or not the commissioner *should perform* TC acknowledgement?